test/common/tests.go

```diff
@@ -28,6 +28,7 @@ var (
 		{"E05 - Verify Grafana Route returns dashboards", TestGrafanaExternalRouteDashboardExist},
 		{"C04 - Verify Alerts exist", TestIntegreatlyAlertsExist},
 		{"C01 - Verify Alerts are not firing apart from DeadMansSwitch", TestIntegreatlyAlertsFiring},
```

Comment:
Can we change this to `Verify Alerts are not pending or firing apart from DeadMansSwitch`